pkg/account/storage/v2/api_key_test.go

```diff
@@ -136,11 +138,13 @@ func TestUpdateAPIKey(t *testing.T) {
 				result := mock.NewMockResult(mockController)
 				result.EXPECT().RowsAffected().Return(int64(1), nil)
 				s.client.(*mock.MockClient).EXPECT().ExecContext(
-					gomock.Any(), gomock.Any(), gomock.Any(),
+					gomock.Any(),
+					gomock.Regex("^UPDATE api_key SET[\\s\\S]*?name\\s*=\\s*\\?[\\s\\S]*?role\\s*=\\s*\\?[\\s\\S]*?disabled\\s*=\\s*\\?[\\s\\S]*?created_at\\s*=\\s*\\?[\\s\\S]*?updated_at\\s*=\\s*\\?[\\s\\S]*?WHERE[\\s\\S]*?id\\s*=\\s*\\?[\\s\\S]*?environment_namespace\\s*=\\s*\\?[\\s\\S]*?"),
```

Comment:
It's a bit hard to read, but it's ok.
Another way to do this would be to replace the placeholders before querying.

For example.

#### Replace placeholders
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/account/storage/v2/account.go#L243-L248

#### SQL
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/account/storage/v2/sql/account_v2/select_accounts_v2.sql#L11

This would require more changes, but it's good because we could reuse the same SQL query file for Postgres for many cases.
MySQL and Postgres have different placeholders.

http://go-database-sql.org/prepared.html
```
MySQL               PostgreSQL            Oracle
=====               ==========            ======
WHERE col = ?       WHERE col = $1        WHERE col = :col
VALUES(?, ?, ?)     VALUES($1, $2, $3)    VALUES(:val1, :val2, :val3)
```